CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 
 - [BUGFIX] Honor the specified log format in the traces subsystem (@mapno)
 
+- [ENHANCEMENT] The agent no longer appends duplicate exemplars. (@tpaschalis)
```

Comment:
Bump this up to the bottom of the enhancements above, generally try to group categories of changes together.

Reply:
Thanks, it's done!